# <center>Pi 上的软件</center>

-----------

本文档介绍 Pi 上的软件。<br>
商业软件需用户自行获取版权并安装。<br>
<br>
![cpu](https://img.shields.io/badge/-cpu-blue) ![gpu](https://img.shields.io/badge/-gpu-green) ![arm](https://img.shields.io/badge/-arm-yellow) ![singularity](https://img.shields.io/badge/-singularity-blueviolet) 标签表明软件有 cpu, gpu, arm 和 singularity 版本


## Pi 上的软件


| Name | Version | Distribution | Introduction |
| ---- | ---- | ---- | ---- |
| [ ABINIT ](https://docs.hpc.sjtu.edu.cn/application/abinit/) | 8.10.3 |![cpu](https://img.shields.io/badge/-cpu-blue)|ABINIT is a package whose main program allows one to find the total energy, charge density and electronic structure of systems made of electrons and nuclei (molecules and periodic solids) within Density Functional Theory (DFT), using pseudopotentials and a planewave or wavelet basis.|
| [ Amber ](https://docs.hpc.sjtu.edu.cn/application/Amber/) ||![cpu](https://img.shields.io/badge/-cpu-blue) ![gpu](https://img.shields.io/badge/-gpu-green)| A package of molecular simulation programs and analysis tools. |
| BCFtools | 1.9 |![cpu](https://img.shields.io/badge/-cpu-blue)| BCFtools is a program for variant calling and manipulating files in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.|
| Bedtools2 | 2.27.1 |![cpu](https://img.shields.io/badge/-cpu-blue)|The bedtools utilities are a swiss-army knife of tools for a wide-range of genomics analysis tasks. The most widely-used tools enable genome arithmetic: that is, set theory on the genome. |
| Bismark  | 0.19.0 |![cpu](https://img.shields.io/badge/-cpu-blue)| Bismark is a program to map bisulfite treated sequencing reads to a genome of interest and perform methylation calls in a single step. The output can be easily imported into a genome viewer, such as SeqMonk, and enables a researcher to analyse the methylation levels of their samples straight away.|
| Bowtie | 1.2.3 |![cpu](https://img.shields.io/badge/-cpu-blue)|Bowtie is an ultrafast, memory-efficient short read aligner geared toward quickly aligning large sets of short DNA sequences (reads) to large genomes.|
| BWA | 0.7.17 |![cpu](https://img.shields.io/badge/-cpu-blue)|BWA is a software package for mapping low-divergent sequences against a large reference genome, such as the human genome.|
| [ CESM ](https://docs.hpc.sjtu.edu.cn/application/CESM/) ||![cpu](https://img.shields.io/badge/-cpu-blue)| Community Earth System Model, or CESM, is a fully-coupled, community, global climate model that provides state-of-the-art computer simulations of the Earth's past, present, and future climate states. |
| CDO | 1.9.8 |![cpu](https://img.shields.io/badge/-cpu-blue)|CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data.|
| CP2K |6.1 |![cpu](https://img.shields.io/badge/-cpu-blue)| A freely available program to perform atomistic and molecular simulations of solid state, liquid, molecular and biological systems. It provides a general framework for different methods such as e.g. density functional theory (DFT) using a mixed Gaussian and plane waves approach (GPW), and classical pair and many-body potentials. |
| Cufflinks | 2.2.1 |![cpu](https://img.shields.io/badge/-cpu-blue)|Cufflinks assembles transcripts, estimates their abundances, and tests for differential expression and regulation in RNA-Seq samples.|
| FastQC | 0.11.7 |![cpu](https://img.shields.io/badge/-cpu-blue)|FastQC aims to provide a simple way to do some quality control checks on raw sequence data coming from high throughput sequencing pipelines.|
| GATK | 3.8 |![cpu](https://img.shields.io/badge/-cpu-blue)| The GATK is the industry standard for identifying SNPs and indels in germline DNA and RNAseq data. |
|Gaussian ||![cpu](https://img.shields.io/badge/-cpu-blue)|Gaussian is a general purpose computational chemistry software package initially released in 1970 by John Pople and his research group at Carnegie Mellon University as Gaussian 70.|
| GMAP-GSNAP | 2019-05-12 |![cpu](https://img.shields.io/badge/-cpu-blue)|GMAP is a tools for rapidly and accurately mapping and aligning cDNA sequences to genomic sequences. GSNAP is designed to align short reads from NGS data and allow detection of short and long range splicing de novo or with a database of know juctions. |
| GraphMap | 0.3.0 |![cpu](https://img.shields.io/badge/-cpu-blue)|A highly sensitive and accurate mapper for long, error-prone reads.|
| [ Gromacs ](https://docs.hpc.sjtu.edu.cn/application/Gromacs/) |2019.2, 2019.4 |![cpu](https://img.shields.io/badge/-cpu-blue) ![gpu](https://img.shields.io/badge/-gpu-green)![arm](https://img.shields.io/badge/-arm-yellow) ![singularity](https://img.shields.io/badge/-singularity-blueviolet) |GROMACS is a versatile package to perform molecular dynamics, i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles. It is primarily designed for biochemical molecules like proteins, lipids and nucleic acids that have a lot of complicated bonded interactions, but since GROMACS is extremely fast at calculating the nonbonded interactions (that usually dominate simulations) many groups are also using it for research on non-biological systems, e.g. polymers. |
| HISAT2 | 2.1.0|![cpu](https://img.shields.io/badge/-cpu-blue) |HISAT2 is a fast and sensitive alignment program for mapping next-generation sequencing reads (both DNA and RNA) to a population of human genomes as well as to a single reference genome. |
| [ LAMMPS ](https://docs.hpc.sjtu.edu.cn/application/Lammps/) | 20190807 |![cpu](https://img.shields.io/badge/-cpu-blue) ![gpu](https://img.shields.io/badge/-gpu-green)![arm](https://img.shields.io/badge/-arm-yellow) ![singularity](https://img.shields.io/badge/-singularity-blueviolet)|(Large-scale Atomic/Molecular Massively Parallel Simulator) a classical molecular dynamics code. |
| LUMPY-SV| 0.2.13 |![cpu](https://img.shields.io/badge/-cpu-blue)| A general probabilistic framework for structural variant discovery.|
| MEGAHIT | 1.1.4| ![cpu](https://img.shields.io/badge/-cpu-blue)|MEGAHIT is an ultra-fast and memory-efficient NGS assembler. It is optimized for metagenomes, but also works well on generic single genome assembly (small or mammalian size) and single-cell assembly.|
| METIS | 5.1.0 |![cpu](https://img.shields.io/badge/-cpu-blue)|METIS is a set of serial programs for partitioning graphs, partitioning finite element meshes, and producing fill reducing orderings for sparse matrices. |
| MrBayes | 3.2.7a |![cpu](https://img.shields.io/badge/-cpu-blue)|MrBayes is a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models.|
| NCBI-RMBlastn | 2.2.28 |![cpu](https://img.shields.io/badge/-cpu-blue)|RMBlast is a RepeatMasker compatible version of the standard NCBI BLAST suite. The primary difference between this distribution and the NCBI distribution is the addition of a new program "rmblastn" for use with RepeatMasker and RepeatModeler.|
|[ Nektar++ ](https://docs.hpc.sjtu.edu.cn/application/Nektar/)|4.4.1 |![cpu](https://img.shields.io/badge/-cpu-blue) |Nektar++ is a spectral/hp element framework designed to support the construction of efficient high-performance scalable solvers for a wide range of partial differential equations (PDE).|
| [ nwChem ](https://docs.hpc.sjtu.edu.cn/application/nwchem/) |6.8.1 |![cpu](https://img.shields.io/badge/-cpu-blue)| NWChem aims to provide its users with computational chemistry tools that are scalable both in their ability to treat large scientific computational chemistry problems efficiently, and in their use of available parallel computing resources from high-performance parallel supercomputers to conventional workstation clusters. The NWChem software can handle: Biomolecules, nanostructures, and solid-state; From quantum to classical, and all combinations; Gaussian basis functions or plane-waves; Scaling from one to thousands of processors; Properties and relativity. | 
|[ Octave ](https://docs.hpc.sjtu.edu.cn/application/Octave/) |5.2.0 |![cpu](https://img.shields.io/badge/-cpu-blue) ![singularity](https://img.shields.io/badge/-singularity-blueviolet) | GNU Octave is software featuring a high-level programming language, primarily intended for numerical computations.|
| [ OpenFoam ](https://docs.hpc.sjtu.edu.cn/application/OpenFoam/) |7, 1712, 1812, 1912| ![cpu](https://img.shields.io/badge/-cpu-blue) ![singularity](https://img.shields.io/badge/-singularity-blueviolet)| OpenFOAM is an open-source toolbox for computational fluid dynamics. OpenFOAM consists of generic tools to simulate complex physics for a variety of fields of interest, from fluid flows involving chemical reactions, turbulence and heat transfer, to solid dynamics, electromagnetism and the pricing of financial options. |
| OVITO ||![cpu](https://img.shields.io/badge/-cpu-blue)| OVITO (Open Visualization Tool) is a scientific visualization and analysis package for atomistic and particle-based simulation data. |
| Paraview |0.4.1 |![cpu](https://img.shields.io/badge/-cpu-blue)| Paraview is a data visualisation and analysis package. Whilst ARCHER compute or login nodes do not have graphics cards installed in them paraview is installed so the visualisation libraries and applications can be used to post-process simulation data. To this end the pvserver application has been installed, along with the paraview libraries and client application. |
| Picard | 2.19.0 |![cpu](https://img.shields.io/badge/-cpu-blue)| Picard is a set of command line tools for manipulating high-throughput sequencing (HTS) data and formats such as SAM/BAM/CRAM and VCF.|
|[ Pytorch ](https://docs.hpc.sjtu.edu.cn/application/Pytorch/) |19.10| ![gpu](https://img.shields.io/badge/-gpu-green) ![singularity](https://img.shields.io/badge/-singularity-blueviolet)| PyTorch is an open source machine learning library based on the Torch library, used for applications such as computer vision and natural language processing, primarily developed by Facebook's AI Research lab. |
| [ Quantum-Espresso ](https://docs.hpc.sjtu.edu.cn/application/Quantum-Espresso/) |6.4.1 |![cpu](https://img.shields.io/badge/-cpu-blue)| Quantum Espresso is an integrated suite of Open-Source computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials. |
|[ R ](https://docs.hpc.sjtu.edu.cn/application/R/)| 1.1.8, 3.6.2 |![cpu](https://img.shields.io/badge/-cpu-blue)|R is a programming language and free software environment for statistical computing and graphics supported by the R Foundation for Statistical Computing. |
|[ Relion ](https://docs.hpc.sjtu.edu.cn/application/Relion/) |3.0.8 |![gpu](https://img.shields.io/badge/-gpu-green)| REgularised LIkelihood OptimisatioN (RELION) employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class averages in electron cryomicroscopy.|
| RNA-SeQC | 1.1.8 |![cpu](https://img.shields.io/badge/-cpu-blue)| RNA-SeQC is a java program which computes a series of quality control metrics for RNA-seq data.|
| Salmon | 0.14.1| ![cpu](https://img.shields.io/badge/-cpu-blue)|Salmon is a tool for wicked-fast transcript quantification from RNA-seq data.|
| SAMtools | 1.9| ![cpu](https://img.shields.io/badge/-cpu-blue)| SAM Tools provide various utilities for manipulating alignments in the SAM format.|
| SOAPdenovo2 | 240| ![cpu](https://img.shields.io/badge/-cpu-blue)| SOAPdenovo is a novel short-read assembly method that can build a de novo draft assembly for the human-sized genomes.|
| SRAtoolkit | 2.9.6 |![cpu](https://img.shields.io/badge/-cpu-blue)|The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for using data in the INSDC Sequence Read Archives.|
| STAR | 2.7.0 |![cpu](https://img.shields.io/badge/-cpu-blue)| Spliced Transcripts Alignment to a Reference (STAR) software is based on a previously undescribed RNA-seq alignment algorithm that uses sequential maximum mappable seed search in uncompressed suffix arrays followed by seed clustering and stitching procedure.|
| [STAR-CCM+](https://docs.hpc.sjtu.edu.cn/application/star-ccm/) |  |![cpu](https://img.shields.io/badge/-cpu-blue)| Much more than just a CFD solver, STAR-CCM+ is an entire engineering process for solving problems involving flow (of fluids or solids), heat transfer and stress.|
| StringTie | 1.3.4d |![cpu](https://img.shields.io/badge/-cpu-blue)| StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts. |
|[ TensorFlow ](https://docs.hpc.sjtu.edu.cn/application/TensorFlow/)|2.0.0 |![gpu](https://img.shields.io/badge/-gpu-green) ![singularity](https://img.shields.io/badge/-singularity-blueviolet)|TensorFlow is a free and open-source software library for dataflow and differentiable programming across a range of tasks. It is a symbolic math library, and is also used for machine learning applications such as neural networks.|
| TopHat | 2.1.2 |![cpu](https://img.shields.io/badge/-cpu-blue)|TopHat is a program that aligns RNA-Seq reads to a genome in order to identify exon-exon splice junctions.|
|VarDictJava |1.5.1|![cpu](https://img.shields.io/badge/-cpu-blue)|VarDictJava is a variant discovery program written in Java and Perl.|
| [ VASP ](https://docs.hpc.sjtu.edu.cn/application/VASP/) ||![cpu](https://img.shields.io/badge/-cpu-blue) ![gpu](https://img.shields.io/badge/-gpu-green)|A package for ab initio, quantum-mechanical, molecular dynamics simulations. |
| VSEARCH | 2.4.3 |![cpu](https://img.shields.io/badge/-cpu-blue) |VSEARCH stands for vectorized search, as the tool takes advantage of parallelism in the form of SIMD vectorization as well as multiple threads to perform accurate alignments at high speed.|
| [ VMD ](https://docs.hpc.sjtu.edu.cn/application/VMD/) |1.9.4 |![cpu](https://img.shields.io/badge/-cpu-blue) ![singularity](https://img.shields.io/badge/-singularity-blueviolet)| VMD is a molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting. |


